validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -139,6 +142,10 @@ public GraffitiProvider getGraffitiProvider() {
     return processor.getFilePairs();
   }
 
+  public boolean isGenerateEarlyAttestations() {
```

Comment:
nit: I'd probably make this just `generateEarlyAttestations` or `shouldGenerateEarlyAttestations` as the `is` prefix doesn't really make sense here (and we don't actually use any JavaBeans so don't need to strictly follow that naming convention).

Reply:
thats just the auto generated name, will change